kubectl-fdb/Readme.md

```diff
@@ -1,9 +1,13 @@
 # Kubectl plugin for FDB on Kubernetes
 
-Current status: *alpha*
-
 ## Installation
 
+Install from release:
+
+```bash
+# Coming soon!
```

Comment:
Is this still "coming soon", or are we prepared to add instructions here?

Reply:
I update this one 👍